kernel/arch/dreamcast/hardware/cdrom.c

```diff
@@ -159,11 +81,11 @@ int cdrom_exec_cmd_timed(int cmd, void *param, int timeout) {
 
     /* Submit the command */
     for(n = 0; n < 10; ++n) {
-        hnd = gdc_req_cmd(cmd, param);
+        hnd = syscall_gdrom_send_command(cmd, param);
         if (hnd != 0) {
             break;
         }
-        gdc_exec_server();
+        syscall_gdrom_run_commands();
```

Comment:
The plural is inappropriate here.
There is always only one command; several cannot be executed simultaneously. This is the processing cycle of the active command for polling registers and changing the status of the command.
It's originally called as ExecServer by Sega devs, I think better keep it as is.

Reply:
I updated the name to syscall_gdrom_exec_server()